src/planscape/stands/management/commands/load_metrics_v2.py

```diff
@@ -0,0 +1,150 @@
+from datetime import datetime, timedelta
+from itertools import repeat
+import multiprocessing
+import time
+from django import db
+from django.contrib.gis.geos import GEOSGeometry
+from django.core.management.base import BaseCommand, CommandParser
+import humanize
+from conditions.models import Condition
+from django.db import connection
+
+from stands.models import Stand, StandMetric
+
+
+def handle_stand_condition(stand_id, condition_id):
+    # stand_id, condition_id = data#
+    with connection.cursor() as cursor:
+        try:
+            cursor.execute(
+                """
+                INSERT INTO stands_standmetric (created_at, min, avg, max, sum, count, condition_id, stand_id) (
+                    SELECT 
+                        now(),
+                        stats.min,
+                        stats.avg,
+                        stats.max,
+                        stats.sum,
+                        stats.count,
+                        stats.condition_id,
+                        stats.stand_id
+                    FROM compute_stand_stats(%s, %s) AS stats
+                    WHERE
+                        stats.condition_id IS NOT NULL AND
+                        stats.count > 0
+                )""",
+                [stand_id, condition_id],
+            )
+        except Exception as ex:
+            print(f"[FAIL] {stand_id} {condition_id} {ex}")
+
+
+class Command(BaseCommand):
+    help = "Loads stand metrics based on conditions existing in the database."
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "--condition-ids",
+            nargs="+",
+            type=int,
+        )
+
+        parser.add_argument(
+            "--size",
+            type=str,
+            default="MEDIUM",
+        )
+
+        parser.add_argument(
+            "--clear",
+            default=True,
+            action="store_true",
+            help="If set to false, the command will not remove previous metrics data from each condition from the raster table.",
+        )
+
+        parser.add_argument("--max-workers", type=int, default=4)
+
+    def clear_data(self, condition_ids=None, size=None):
+        filters = {}
+        if condition_ids:
+            filters["condition_id__in"] = condition_ids
+        if size:
+            filters["stand__size"] = size
+        delete = StandMetric.objects.filter(**filters).delete()
+        self.stdout.write(f"Deleted data {delete}")
+
+    def get_condition_extent(self, condition_id):
+        with connection.cursor() as cursor:
+            cursor.execute(
+                """
+                SELECT
+                    ST_Envelope(ST_Collect(ST_Envelope(raster))) AS "geometry"
```

Comment:
In the SQL version of this code, you wrapped this with ST_Transform(..., 4269). Is that no longer needed here?